### DISCUSSION TITLE
"tab.activeBorder"

### DISCUSSION THREAD

**Discussion by Napolite** [zed-industries/zed#15753] [Feature Requests]
How do I get the equivalent of tab.activeBorder in the  zed theme.json? I have looked through most themes and I can't find something like it.

Also, is there a way to set the color of active tab text?

Also is there a theme development documentation somewhere that I can read?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
